scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontendState.java

```diff
@@ -288,13 +288,9 @@ synchronized SchedulerState getState(boolean myJobsOnly)
 
         ListeningUser ui = identifications
                 .get(PAActiveObject.getContext().getCurrentRequest().getSourceBodyID());
-        try {
-            checkOwnStatePermission(myJobsOnly, ui.getUser());
-            return myJobsOnly ? sState.filterOnUser(ui.getUser().getUsername()) : sState;
-        } catch (PermissionException ex) {
-            logger.info(ex.getMessage());
-            throw ex;
-        }
+
```

Comment:
I'm afraid, that by this modification, a user will be able to see all other users jobs (task list)